spec/components/auth/default_current_user_provider_spec.rb

```diff
@@ -586,6 +586,38 @@ def provider(url, opts = nil)
     expect(UserAuthToken.where(user_id: user.id).count).to eq(2)
   end
 
+  it "logging on user cleans up old sessions" do
+    user = Fabricate(:user)
+
+    yesterday = Time.zone.now - 1.day
+
+    UserAuthToken.insert_all((1..(UserAuthToken::MAX_SESSION_COUNT + 2)).to_a.map do |i|
+      {
+        user_id: user.id,
+        created_at: yesterday + i.seconds,
+        updated_at: yesterday + i.seconds,
+        rotated_at: yesterday + i.seconds,
+        prev_auth_token: "abc#{i}",
+        auth_token: "abc#{i}"
+      }
+    end)
+
+    # Has 2 more sessions than allowed
+    expect(UserAuthToken.where(user_id: user.id).count).to eq(UserAuthToken::MAX_SESSION_COUNT + 2)
```

Comment:
Assuming `insert_all` is tested elsewhere, I don't think you need to test that the Auth Tokens are created successfully by this test.